src/ontology/nbo-edit.owl

```diff
@@ -1469,15 +1653,11 @@ AnnotationAssertion(rdfs:comment obo:NBO_0000033 "Carpoptosis is most often caus
 AnnotationAssertion(rdfs:label obo:NBO_0000033 "carpoptosis")
 SubClassOf(obo:NBO_0000033 obo:NBO_0000032)
 
-# Class: obo:NBO_0000034 (sexual activity)
+# Class: obo:NBO_0000034 (mating behavior)
```

Comment:
you rename sexual activity to mating behaviour and the other way around? This seems risky

Reply:
Yes indeed - this is discussed in the remarks (IAO:0000116) for both entries. I may be wrong on at least four counts:

1. I may miscast the language use in Biology for these terms as 'contradictory' instead of it simply being my misunderstanding: I need other Biologists to weigh in, but I think the terms are important and contradict usage in the mainstream of Biology
2. Where a prior definition and associated information (e.g. parent/child relations) were sufficiently ambiguous, a tightened definition and improved relational links need not require obsoletion and a new ID if they did not contradict any of the original
3. Ontological definitions can also be somewhat creative if they enable effective categorisation. (For example, we have stuck with the idea NBO started with, that parenthood is excluded from 'reproductive behavior' despite the biological mainstream, because it is pragmatic and too disruptive to change.) Introducing the role of reproductive organs to 'mating behavior' is a potentially useful characterisation in our context, which may not be uppermost in biological minds, but potentially helps with our ontology structure
4. There may be another ontological problem with swapping the labels, and I bow to the insights of any Ontologist on this, but I understand that:
- the original 'things' are essentially the same as before, just the labels need changing
- even if we were to obsolete either existing 'thing'/ID, we could still reuse its old label for a new 'thing'/ID, with an effectively identical outcome for NBO but more problems for other users

Reply:
This will be more easily available for discussion when the pull request is merged.

Reply:
This PR will not be merged until after this discussion - merging to master means its approved in its entirety!  I am sure there will be some pushback and we need to give the community a chance to intervene!

Reply:
Sorry - mixed up my terminology! I'm guessing its hypoxia from all the coughing of my recent chest infection ;-) All along I have meant **commit** not **pull request**. Now that the repro-dict branch nbo-edit file contains the proposed changes we can begin the content discussion. I'm unresolving this conversation so this specific issue can be discussed. It would be good if others felt able to open up other conversations as well as comment on this one.
@matentzn @pmidford @aclark-binghamton-edu @jannahastings @dosumis